s3parq/publish_parq.py

```diff
@@ -203,13 +203,15 @@ def log_size_estimate(num_bytes):
 """
     logger.debug(batch_log_message)
     if ideal_size > frame_size_est:
-        return tuple([dataframe])
+        yield {'lower':0,'upper':len(dataframe)}
```

Comment:
yeah this makes much more sense given the whole point of sizing the frames is to scale